roles/unicorn/tasks/main.yml

```diff
@@ -17,6 +17,7 @@
 
 - name: enable unicorn at boot
   service:
+    daemon_reload=yes # Ensure unicorn init script is reloaded
     name=unicorn_{{ app }}
     enabled=yes
```

Comment:
Actually I don't think this is the right way: 
* when running the first time, systemd would load the new script anyway.
* when running a subsequent time, the service is already enabled so this has no effect. 

It really needs to be a notify on the command that updates the file. But as we don't support running subsequent times I won't bother trying to do that, I'll just add a comment..